PowerEditor/installer/nativeLang/english.xml

```diff
@@ -1310,7 +1310,8 @@ You can re-activate this dialog in Preferences later."/>
                 <Item id="4" name="Always Yes"/>
             </DoSaveAll> <!-- HowToReproduce: Check the 'Enable Save All confirm dialog' checkbox in Preference->MISC, now click 'Save all' -->
         </Dialog>
-        <MessageBox> <!-- $INT_REPLACE$ is a place holder, don't translate it -->
+        <MessageBox>
+            <!-- $INT_REPLACE$ and $STR_REPLACE$ are a place holders, don't translate these place holders -->
```

Comment:
Also just comment:
$INT_REPLACE$ and $STR_REPLACE$ are a place holders -> $INT_REPLACE$ and $STR_REPLACE$ are place holders